algorithms/src/snark/varuna/ahp/selectors.rs

```diff
@@ -85,7 +85,7 @@ pub(crate) fn apply_randomized_selector<F: PrimeField>(
     // This removes a mul and div by v_H operation over each circuit's (target_domain - src_domain)
```

Comment:
@vicsn can you confirm the comment here remains correct?

Reply:
Confirmed